### PR TITLE
Update error.md CLIENT_FETCH_ERROR Section

### PR DESCRIPTION
**Pull request by ScribbleTons**
This additional description will help other developers debug next-auth configuration error for when they have the appropriate configuration but stale cookies on their browser.

> _NOTE_:
>
> - It's a good idea to open an issue first to discuss potential changes.
> - Please make sure that you are _NOT_ opening a PR to fix a potential security vulnerability. Instead, please follow the [Security guidelines](../Security.md) to disclose the issue to us confidentially.

## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 🎫 Affected issues

Please [scout and link issues](https://github.com/nextauthjs/next-auth/issues) that might be solved by this PR.

Fixes: INSERT_ISSUE_LINK_HERE

## 📌 Resources

- [Security guidelines](../Security.md)
- [Contributing guidelines](../CONTRIBUTING.md)
- [Code of conduct](../CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: docs/docs/errors.md [hunk 1]
```diff
@@ -182,4 +184,11 @@ Useful links:
 - https://next-auth.js.org/configuration/nextjs#pages
 - https://next-auth.js.org/configuration/pages
 - https://nextjs.org/docs/advanced-features/middleware#matcher
+
+---
+
+#### Troubleshooting
+###### CLIENT_FETCH_ERROR
+Can also be caused by stale cookies and other configuration errors. To fix, open your browser devtools and clear your cookies from `application` tab for dev website e.g localhost:3000.
```
Comment (ndom91) [inline docs/docs/errors.md:192]:
```suggestion
Can also be caused by stale cookies and other configuration errors. To fix the stale cookies issue, open your browser devtools for your dev app, go to the `Application` tab and click "Clear Storage". 
```

File: docs/docs/errors.md [hunk 2]
```diff
@@ -182,4 +184,11 @@ Useful links:
 - https://next-auth.js.org/configuration/nextjs#pages
 - https://next-auth.js.org/configuration/pages
 - https://nextjs.org/docs/advanced-features/middleware#matcher
+
+---
+
+#### Troubleshooting
+###### CLIENT_FETCH_ERROR
+Can also be caused by stale cookies and other configuration errors. To fix, open your browser devtools and clear your cookies from `application` tab for dev website e.g localhost:3000. 
+Here's a useful link to a stackoverflow [solution](https://stackoverflow.com/questions/73564345/next-auth-credentials-provider-server-error) on similar problem.
```
Comment (ndom91) [inline docs/docs/errors.md:193]:
```suggestion
For more information, check out this [stackoverflow](https://stackoverflow.com/questions/73564345/next-auth-credentials-provider-server-error).
```

**Review by ndom91 — COMMENTED** [review]
Made some copy suggestions, but generally LGTM

**Comment by balazsorban44** [comment]
@ScribbleTons could you open a bug report with a reproduction where this error caused you stale cookie issues? I would consider this case a bug on our side that would need to be addressed. But we already have a cleanup in place when the session endpoint fails, so I would like to dig deeper. See:

https://github.com/nextauthjs/next-auth/blob/v4/packages/next-auth/src/core/routes/session.ts#L159

Also, server configuration errors should have more specific errors logs for them on server-side, so I don't think that needs to be mentioned.

TLDR; could you create a minimal public reproduction with the behavior that caused you this error?

**Comment by balazsorban44** [comment]
Closing as per https://github.com/nextauthjs/next-auth/pull/6147#issuecomment-1363415350, it's currently unclear what this PR tries to clarify as there is no reproducible bug. Let's create an issue, then we can reopen, if verified.

**Pull request closed without merging**